infra/resources/azuredeploy.json

```diff
@@ -9,30 +9,30 @@
             "type": "string"
         },
         "docdb_name": {
-            "defaultValue": "acm-notes-docdb",
+            "defaultValue": "notes-app-docdb",
             "type": "string"
         },
         "eventhub_namespace_name": {
-            "defaultValue": "acm-notes-hub",
+            "defaultValue": "notes-app-hub",
             "type": "string"
         },
         "registry_name": {
-            "defaultValue": "acmnotesregistry",
+            "defaultValue": "notesappregistry",
```

Comment:
In some you use hyphens and others you don't. Minor issue though, not necessary to change if there's a reason behind it.

Reply:
There is a reason behind this. For some reason, Storage containers and Container registries (my guess is that they are both really just storage containers), you cannot have hyphens in the names. But it looks nicer with the hyphens for literally everything else...